stable/datadog/values.yaml

```
@@ -665,6 +666,65 @@ agents:
     #
     serviceAccountName: default
 
+  ## @param podSecurity - object - optional
+  ## Provide Daemonset PodSecurityPolicy configuration
+  podSecurity:
+
+    ## @param podSecurityPolicy - object - required
+    ## Provide Daemonset PodSecurityPolicy configuration
+    podSecurityPolicy:
+
+      ## @param created - boolean - optional
+      ## If true, create a PodSecurityPolicy resource for Agent pods
+      #
+      create: false
+
+    ## @param securityContextConstraints - object - required
+    ## Provide Daemonset securityContextConstraints configuration
+    securityContextConstraints:
+
+      ## @param created - boolean - optional
+      ## If true, create a SecurityContextConstraints resource for Agent pods
+      #
+      create: false
+
+    ## @param privileged - boolean - optional
+    ## If true, Allow to run privileged containers
+    #
+    privileged: false
+
+    ## @param capabilites - list - optional
+    ## Allowed capabilites
+    #
+    capabilites:
+    - SYS_ADMIN
+    - SYS_RESOURCE
+    - SYS_PTRACE
+    - NET_ADMIN
+    - IPC_LOCK
+
+    ## @param volumes - list - optional
+    ## Allowed volumes types
+    #
+    volumes:
+    - configMap
+    - downwardAPI
+    - emptyDir
+    - hostPath
+    - secret
+
+    ## @param seccompProfiles - list - optional
+    ## Allowed seccomp profiles
+    #
+    seccompProfiles:
+    - "*"
```

Do we want do restrict the default list of allowed seccomp profiles ?

```suggestion
    - "runtime/default"
    - "localhost/system-probe"
```

We should also add to this list `{{ .Values.datadog.system-probe.seccomp }}`, but I’m not sure this is possible inside `values.yaml`.

---

good idea